keyboards/dz60/config.h

```diff
@@ -25,7 +25,7 @@
 
 /* number of backlight levels */
 #define BACKLIGHT_PIN B6
-#define BACKLIGHT_LEVELS 5
+#define BACKLIGHT_LEVELS 10
```

Comment:
This shouldn't be changed here. 

In your keymaps `config.h` file, you can add the following instead:
```
#undef BACKLIGHT_LEVELS
#define BACKLIGHT_LEVELS 10
```

This will change the value without having to mess with core keyboard files. 
(I actually do this to change the display name of my keyboards)

Reply:
sure, sounds good!